base_external/Config.in

```diff
@@ -0,0 +1 @@
+#source "$BR2_EXTERNAL_assignment_4_PATH/package/aesd-assignments/Config.in"
```

Comment:
@VatsalSheth this looks wrong, does this work? I'd expect this needs to match external.desc definition

Reply:
@dwalkes Package was not included that's why maybe. But I have removed the package now and pushed the changes.